.github/workflows/test-deps.yml

```diff
@@ -7,14 +7,13 @@ on:
   workflow_dispatch:
 
 jobs:
-    test-nodevdeps:
+    test-latest-nodevdeps:
```

Comment:
So these names have gotten complicated enough that perhaps it's time for a comment at the top describing all the targets?